frappe/public/js/frappe/form/controls/button.js

```diff
@@ -7,7 +7,9 @@ frappe.ui.form.ControlButton = class ControlButton extends frappe.ui.form.Contro
 		var me = this;
 		const btn_type = this.df.primary ? "btn-primary" : "btn-default";
 		const btn_size = this.df.btn_size ? `btn-${this.df.btn_size}` : "btn-xs";
-		this.$input = $(`<button class="btn ${btn_size} ${btn_type}">`)
+		this.$input = $(
+			`<button class="btn ${btn_size} ${btn_type} ellipsis" title="${this.df.label}">`
```

Comment:
The label should be escaped in the string interpolation, or set using .attr("title", ...)